packages/core/src/sdk/cart/index.ts

```diff
@@ -174,14 +174,20 @@ export const useCart = () => {
         (acc, curr) => acc + (isGift(curr) ? 0 : curr.quantity),
         0
       ),
-      total: cart.items.reduce(
-        (acc, curr) => acc + curr.price * curr.quantity,
-        0
-      ),
-      subTotal: cart.items.reduce(
-        (acc, curr) => acc + curr.listPrice * curr.quantity,
-        0
-      ),
+      total:
+        isValidating && !optimisticCart
```

Comment:
This is a nitpick, but I'd put the fixed value first and the variable value later.
```suggestion
       !optimisticCart && isValidating
```